content/docs/capabilities/authentication.mdx

```diff
@@ -54,6 +54,10 @@ By configuring your applications to route requests to Pomerium’s Proxy service
 
 ## External data sources (Enterprise)
 
+:::enterprise Pomerium Enterprise
```

Comment:
side note: now that we have a custom component for this admonition, we may want to implement a default title in it, so we wouldn't need to repeat "Pomerium Enterprise" each time we use it

Reply:
Based on their example, the title must be manually entered: https://docusaurus.io/docs/markdown-features/admonitions#custom-admonition-type-components. If it's possible to add a default title, we can do that for sure.